<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>instanceSettingsWidget</class>
 <widget class="QWidget" name="instanceSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>332</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>332</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>332</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QLineEdit" name="apiIdEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>API ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="apiHashEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>API HASH</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ownerIdEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Your ID</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>34</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>API ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>74</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>API HASH:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>114</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Your ID:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="runInstanceButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Advanced settings</string>
   </property>
   <widget class="QComboBox" name="loggingLevelCB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string notr="true">INFO</string>
    </property>
    <item>
     <property name="text">
      <string notr="true">INFO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">DEBUG</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="consoleLoggingLevelCB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string notr="true">As logging level</string>
    </property>
    <item>
     <property name="text">
      <string>As logging level</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">INFO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">DEBUG</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="enablePLAutoUpdateChB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable plugins Auto-Update</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="removeInstanceButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Instance</string>
   </property>
  </widget>
  <widget class="QListWidget" name="instancesList">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>331</height>
    </rect>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::SinglePass</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="newInstanceButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>New instance</string>
   </property>
  </widget>
  <widget class="QLabel" name="instanceName">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;</string>
   </property>
   <property name="text">
    <string>No instance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveSettingsButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save settings</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
